pkgs/development/python-modules/gfsubsets/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, importlib-resources
+, setuptools
+, setuptools-scm
+, youseedee
+}:
+
+buildPythonPackage rec {
+  pname = "gfsubsets";
+  version = "2024.2.5";
+  gitTag = "v2024.02.05";
```

If you want to leak this into the environment, change it to

```suggestion
  env.gitTag = "v2024.02.05";
```

Otherwise, I don't see any usage, please remove it.

---

It's used below in `meta.changelog`:

```
changelog = "https://github.com/googlefonts/nam-files/releases/tag/${gitTag}";
```

Unfortunately because of upstream's chosen versioning scheme, the git tag and PyPI versions don't match. I didn't want to try and craft some complicated expression to predict the git tag from the PyPI version or vice versa, I was intending to just use this variable here as a reminder to keep these two versions in sync with each other in future.

Admittedly, that means the normal Python package update script won't work correctly on this package. I'm not sure what the best solution is in this case.

Given that the release notes on Github contain very little useful information, maybe the simplest option is to just leave `meta.changelog` unspecified?